scripts/bootstrap

```diff
@@ -11,7 +11,7 @@ set -e
 script_path=$(dirname "$0")
 
 # Install application code and entrypoints from 'setup.py'
-pip install --use-deprecated=legacy-resolver -e $script_path/..[all]
+pip install -e $script_path/..[all]
```

Comment:
maybe install before this line the explicit pip version, like we do in the ./run-tests.sh, otherwise if someone has newest pip, it will hang trying to install dependencies (unless you tested the newest and there is no problem)

Reply:
I just tested it with the latest pip (`pip 21.0.1`) and there are no problems